src/SPC/builder/unix/UnixBuilderBase.php

```diff
@@ -264,22 +265,70 @@ protected function patchPhpScripts(): void
         }
     }
 
+    /**
+     * Process the --with-frankenphp-app option
+     * Creates app.tar and app.checksum in source/frankenphp directory
+     */
+    protected function processFrankenphpApp(): void
```

Comment:
How about processing frankenphp app before building everything? Although I don't have any build requirements for frankenphp embed app, but if something goes wrong here, it would be annoying to wait for dozens of minutes to build and then report an error.